dev-libs/ivykis/ivykis-0.42.3-r1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Library for asynchronous I/O readiness notification"
+HOMEPAGE="https://github.com/buytenh/ivykis"
+SRC_URI="https://github.com/buytenh/ivykis/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="static-libs"
+
+src_configure() {
+	econf $(use_enable static-libs static)
+}
+
+src_install() {
+	default
+
+	if ! use static-libs; then
+		find "${ED}" -name "*.la" -delete || die
```

Comment:
Do you really need the .la file when .pc file is supplied?

Reply:
(and there are no dependencies apparently)

Reply:
Yes, by reading https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives it really seems "*.la" can go, I'll adjust, thanks.